src/components/Events.tsx

```diff
@@ -20,14 +22,25 @@ export const Events: React.FC = () => {
     fetchEvents();
   }, []);
 
+  const handleRedirect = (event: Event) => {
+    router.push({
+      pathname: `/event/${event.id}`,
+    });
+  };
```

Comment:
Like 80% of places, you would see exactly this approach. Where you imperatively would navigate using router.push. Problem is, it is not an anchor, which creates multiple issues
- you cannot click on it via middle button
- you cannot "open in another tab"
- It does not hint the user where he will be taken (at the bottom left of the screen).

The best thing would be to have `<li><Link ...` with the path. But in our case. Keep this. We are fine. I just want you to know

It would possible would be stellar code in majority of reviews you would encounter

Reply:
understood